### PR TITLE
cb_channels: Call to target device and intercept given UUID to it, after answer.

### PR DESCRIPTION
**Pull request by skorobkov**
This combination of `quickcall` and `call pickup` for `cb_channels`.
For now its only call to one device, `user` target will be added later.

### CODE REVIEW

**Comment by lazedo** [comment]
@skorobkov hey, this is cool! some thoughts.
originate+intercept is more like  `move` and there's currently an implementation available in konami/ konami_move action.

i know its not available in cb_channels but, the other implementations (`callflow`, `hangup`, `transfer`) follow the pattern of collecting the information and publish a metaflow request which will be listened by konami and execute the action.

this way, a new action will be available for both channels api and dtmf bindings. 

about the formatting, you may want to run "make fmt"

Best

**Comment by skorobkov** [comment]
Ok. I will try rewrite to metaflow/konami.

**Comment by lazedo** [comment]
@skorobkov great. let us know if there's anything we can help you with.

**Comment by skorobkov** [comment]
Problem with `konami_move` it's bridge **opposite leg** of given Call-ID. And it wont work in scenario where is no other leg (ex. parked call).
What i should do? Modify konami_move (add some special paramaters in `data`)? Or create konami_intercept (based on konami_move)?

**Comment by skorobkov** [comment]
Trying use konami_move.
Adding paramter `intercept_logic:`true` to "data" will swap channels in chosing which one is should be moved.

**Comment by lazedo** [comment]
@skorobkov Hi,  the `konami_intercept` module seems to be missing ?

**Comment by skorobkov** [comment]
:sweat:

**Comment by skorobkov** [comment]
Rebased to master & squashed.

File: applications/konami/src/module/konami_intercept.erl [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+-spec build_originate(kz_json:objects(), ne_binary(), kapps_call:call()) -> kz_proplist().
+build_originate([], _CallId, _Call) -> [];
+build_originate(Endpoints, CallId, Call) ->
+    lager:debug("targeting ~s for intercept", [CallId]),
+    props:filter_undefined(
+      [{<<"Application-Name">>, <<"bridge">>}
+      ,{<<"Endpoints">>, Endpoints}
+      ,{<<"Existing-Call-ID">>, CallId}
+      ,{<<"Intercept-Unbridged-Only">>, 'false'}
```
Comment (lazedo) [inline applications/konami/src/module/konami_intercept.erl]: @skorobkov this should be `true` for an `intercept` action or at least fetched from Data (cb_channels should get from params to build it)

**Comment by lazedo** [comment]
@skorobkov thanks for this. please note that we may refactor the `build_endpoints` and `build_originate` in order to de-duplicate code existing in `konami_move` and `konami_intercept`

**Pull request merged**